app/views/projects/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Projects</h1>
+<ul>
+  <% @projects.each do |project| %>
+    <p>Project Name: <a href="<%= project_path(project) %>"><%= project.name %></a></h3>
```

Comment:
Similarly, try to use `link_to` as a view helper rather than the html `<a>` tag.